aws/components/datafeed/setup.ftl

```diff
@@ -238,15 +238,23 @@
                                                 cmkKeyId,
                                                 streamBackupLoggingConfiguration )]
 
+        [#-- Establish bucket prefixes --]
+        [#local streamS3DestinationPrefix = solution.Bucket.Prefix]
+        [#local streamS3DestinationErrorPrefix = solution.Bucket.ErrorPrefix]
 
         [#switch (destinationLink.Core.Type)!"notfound" ]
             [#case BASELINE_DATA_COMPONENT_TYPE]
+                [#if destinationLink.Core.SubComponent.Name == "opsdata"]
+                    [#local streamS3DestinationPrefix = formatPath(true, core.FullAbsolutePath, solution.Bucket.Prefix)]
+                    [#local streamS3DestinationErrorPrefix = formatPath(true, core.FullAbsolutePath, solution.Bucket.ErrorPrefix)]
```

Comment:
Other way around 

So the prefix defined on the Datafeed would define what type of data it is, CWLogs for example

Reply:
Gotcha, fixed up in ff090c0